src/Microsoft.AspNet.Mvc.Razor/RazorViewEngineOptions.cs

```diff
@@ -65,5 +68,22 @@ public string Configuration
                 _configuration = value;
             }
         }
+
+        /// <summary>
+        /// Gets or sets the callback that is used to customize Razor compilation
```

Comment:
`/// Customizations made here would not reflect in tooling (Intellisense)` sound ok?